src/steps/03-Background/CurrentEnvironment/InstanceDetails.vue

```diff
@@ -525,6 +527,36 @@ export default class InstanceDetails extends Mixins(SaveOnLeave) {
     return isValid;
   }
 
+  /**
+   * Compiles and returns the sequence number for pricing details section
+   */
+  getPricingDetailsSequenceNum(): string {
+    if (this.hasTellUsAboutInstanceHeading) {
+      return "4.";
+    } else {
+      return "3.";
+    }
+  }
+
+  /**
+   * Compiles and returns the sequence number for additional information section
+   */
+  getAdditionalInfoSequenceNum(): string {
+    if (this.hasTellUsAboutInstanceHeading) {
+      if (this.instanceData.instance_location === "ON_PREM") {
```

Comment:
ternary please for this and in the `else` block below 
`return this.instanceData.instance_location === "ON_PREM" ? "4." : "5.";`